test/apiv2/26-containersWait.at

```diff
@@ -21,29 +18,20 @@ t POST "containers/${CTR}/wait?condition=non-existent-cond" 400
 
 t POST "containers/${CTR}/wait?condition=not-running" 200
 
+(sleep 2;podman start "${CTR}") &
+child_pid=$!
+
+# This will block until the background job completes
 t POST "containers/${CTR}/wait?condition=next-exit" 200 \
   .StatusCode=0 \
-  .Error=null &
-child_pid=$!
-podman start "${CTR}"
+  .Error=null
 wait "${child_pid}"
 
 
-# check if headers are sent in advance before body
-WAIT_TEST_ERROR=""
-curl -I -X POST "http://$HOST:$PORT/containers/${CTR}/wait?condition=next-exit" &> "/dev/null" &
+(sleep 2;podman container rm "${CTR}") &
 child_pid=$!
-sleep 0.5
-if kill -2 "${child_pid}" 2> "/dev/null"; then
-  echo -e "${red}NOK: Failed to get response headers immediately.${nc}"  1>&2;
-  WAIT_TEST_ERROR="1"
-fi
 
```

Comment:
I have no idea what this test was supposed to do, so I simply removed it.

Reply:
I think it's testing that the response to a Wait request sends HTTP headers immediately, but does not finish writing/close the connection until the container actually exits.

Reply:
That's what the comment says, but not (AFAICS) what the code does...?

Reply:
UPDATE: I fixed it.